languages/csharp/exercises/concept/object-initialization/.docs/after.md

```diff
@@ -0,0 +1,41 @@
+Object initializers are an alternative to constructors. The syntax is a comma separated list of field name=value pairs, illustrated below:
+
+```csharp
+public class Person
+{
+    public string Name;
+    public string Address;
+}
+
+...
+
+var person = new Person{Name="The President", Address = "Élysée Palace"};
+```
+
+Collections can also be initialized in this way. Typically, this is accomplished with comma separated lists as shown here:
```

Comment:
Should we mention that this works for any type that implements `IEnumerable` and an appropriately typed `Add` method?

Reply:
Good call

Reply:
One of the rare examples of duck-typing being using in C# :)

Reply:
As I believe foreach is as well

Reply:
Correct. And `async`